enterprise/app/trends/drilldown_page.tsx

```diff
@@ -864,17 +863,7 @@ export default class DrilldownPageComponent extends React.Component<Props, State
                     <div className="history">
                       <div className="container nopadding-dense">
                         {this.state.eventData.invocations.map((invocation) => (
-                          <a
-                            href={`/invocation/${invocation.invocationId}`}
-                            onClick={(e) => {
-                              // TODO(siggisim): Switch this to using the <Link> component
-                              if (e.metaKey || e.ctrlKey) {
-                                return;
-                              }
-                              e.preventDefault();
-                            }}>
-                            <HistoryInvocationCardComponent invocation={invocation} />
-                          </a>
```

Comment:
In a previous PR, I made the `<HistoryInvocationCardComponent>` itself be a link, so the wrapping link is no longer needed.